CMakeLists.txt

```diff
@@ -61,9 +62,35 @@ FILE(GLOB_RECURSE DLR_INC
     "${TREELITE_SRC}/include/*.h"
 )
 
+if(USE_OPENCL)
+    message("USING OpenCL")
+    if(USE_OPENCL STREQUAL "ON")
+      set(USE_OPENCL "/opt/intel/opencl")
```

Comment:
Should we have an (environment) variable for the paths instead of hard code them here?

Reply:
Fixed in new commit, can you check?